contracts/bancor-portal/BancorPortal.sol

```diff
@@ -117,6 +121,7 @@ contract BancorPortal is IBancorPortal, ReentrancyGuardUpgradeable, Utils, Upgra
         validAddress(address(uniswapV2Factory))
         validAddress(address(sushiSwapV2Router))
         validAddress(address(sushiSwapV2Factory))
+        validAddress(address(weth))
```

Comment:
As it stands, no need to cast `weth` to an address.

Reply:
outdated